packages/server/tests/clients/servicesClient.ts

```diff
@@ -75,7 +75,7 @@ export default class ServicesClient {
     this.network = network;
 
     if (!network) network = '{}';
-    const opPrivateKey = PrivateKey.fromString(key);
+    const opPrivateKey = PrivateKey.fromStringECDSA(key);
```

Comment:
Would this work if the key is ED2? Should we make this dynamically accept any type of key?

Reply:
We decided to go with HEX ECDSA for testing.

Reply:
Updated with supporting code for all formats.